cmd/kubelet/app/options/options.go

```diff
@@ -441,6 +433,9 @@ func AddKubeletConfigFlags(mainfs *pflag.FlagSet, c *kubeletconfig.KubeletConfig
 	fs.DurationVar(&c.Authorization.Webhook.CacheUnauthorizedTTL.Duration, "authorization-webhook-cache-unauthorized-ttl", c.Authorization.Webhook.CacheUnauthorizedTTL.Duration, ""+
 		"The duration to cache 'unauthorized' responses from the webhook authorizer.")
 
+	fs.StringVar(&c.CertDirectory, "cert-dir", c.CertDirectory, "The directory where the TLS certs are located. "+
+		"If --tls-cert-file and --tls-private-key-file are specified by the Kubelet's --config flag (see https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information), this option will be ignored.")
```

Comment:
fields have a different names in config, without dashes

```suggestion
		"If tlsCertFile and tlsPrivateKeyFile are specified by the Kubelet's --config flag (see https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information), this option will be ignored.")
```

Reply:
Out of curiosity, this should also change right?

https://github.com/kubernetes/kubernetes/pull/104316/files#diff-f38ee9f130deb5f4bcd61d857729306f8416f7810ee28a21f20d954f9f4faba3R441

Reply:
flags are still supported, right? Maybe just add "... or corresponding kubelet config flags."

Reply:
Got it! thanks! I got myself confused. I have taken in your suggestion as it is. thanks for helping.